RoddyCore/src/de/dkfz/roddy/Roddy.java

```diff
@@ -590,12 +591,22 @@ public static void initializeJobManager(boolean fullSetup) throws ClassNotFoundE
             return;
 
         ClassLoader classLoader;
-        String jobManagerClassID;
-        Class jobManagerClass;
+        String jobManagerClassID = "";
+        Class jobManagerClass = null;
 
-        classLoader = LibrariesFactory.getGroovyClassLoader();
-        jobManagerClassID = Roddy.getApplicationProperty(Constants.APP_PROPERTY_COMMAND_FACTORY_CLASS, PBSJobManager.class.getName());
-        jobManagerClass = classLoader.loadClass(jobManagerClassID);
+        try {
+            classLoader = LibrariesFactory.getGroovyClassLoader();
+            jobManagerClassID = Roddy.getApplicationProperty(Constants.APP_PROPERTY_COMMAND_FACTORY_CLASS);
+            if(RoddyConversionHelperMethods.isNullOrEmpty(jobManagerClassID)) jobManagerClassID = "UNSET";
+            jobManagerClass = classLoader.loadClass(jobManagerClassID);
```

Comment:
The error message uses the class name for its output. UNSET Will result in:
logger.severe("Could not find job manager class: UNSET, available are: ...

Think that's ok or?